src/components/Profile.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import styled from "styled-components";
+
+export default function Profile({ profile }) {
+    const { id, name, age, role, univ, major, phoneNum, email, githubLink, imageUrl } = profile;
+
+    return (
+        <ProfileCard>
+            <Info>
+                <Detail>
+                    <div style={{
+                        fontSize: '2rem',
+                        fontWeight: 'bold',
+                        marginBottom: '1rem'
+                    }}>{name}({age})</div>
+                    <div style={{
+                        fontSize: '1.4rem',
+                        fontWeight: 'bold'
+                    }}>{role}</div>
+                    <div style={{
+                        fontSize: '1.4rem',
+                    }}>{major}</div>
+                    <div style={{ flex: 1 }}></div>
+                    <div style={{
+                        fontSize: '1.4rem',
+                        color: 'rgb(0, 0, 153)'
+                    }}>{phoneNum}</div>
+                    <div style={{
+                        fontSize: '1.4rem',
+                        color: 'rgb(0, 0, 153)'
+                    }}>{email}</div>
+                    <a style={{ fontSize: '1.4rem' }} href={githubLink}>{githubLink}</a>
+                </Detail>
+                <img style={{ width: '20rem', height: '20rem' }} src={imageUrl} />
```

Comment:
아래 정의하신 ProfileImage 써주시면 좋을 것 같아요!
```suggestion
                <ProfileImage src={imageUrl} />
```